server/app/views/applicant/ApplicantProgramFileUploadBlockEditTemplate.html

```diff
@@ -124,6 +125,9 @@
           </div>
         </div>
       </div>
+      <th:block th:each="footerTag : ${fileUploadFooterTags}">
+        [(${footerTag.render()})]
+      </th:block>
```

Comment:
I don't feel strongly here, but I wonder if we can instead insert the header with a `th:include` and then insert the new scripts there, so we don't have scripts randomly in the `<body>`, like this:

```
<head th:include="~{applicant/ApplicantBaseFragment :: pageHeaderScriptsAndLinks}">
    <th:block th:each="footerTag : ${fileUploadFooterTags}">
        [(${footerTag.render()})]
    </th:block>
</head>
```